content/docs/components/introduction.md

```diff
@@ -97,7 +112,7 @@ See also: [Slots reference](./slots.md)
 The contents of the `main` slot are place between the opening and the closing tags.
 
 ```edge
-@component('components/button', {
+@!component('components/button', {
```

Comment:
This is wrong. This tag is closed separately hence no self closing `!` is needed.

Reply:
I do understand the difference between self-closing tags (e.g., @!component) and separately-closed tags (e.g., @component ... @end), but it seems that some sections of the documentation do not reflect this distinction, which is why I submitted my pull request.

For instance, in the current documentation, we have examples like the following:

`
@component('components/button', {
  type: 'submit',
  class: 'btn btn-large',
  text: 'Login'
})

@input({
  type: 'text',
  name: 'title',
  id: 'title',
})
`

However, this leads to an "Unclosed tag slot" error.

Additionally, there's another case where the snippet below results in both an "Unclosed tag slot" error and an "Unexpected token" error:
`
@component('components/button', {
  type: 'submit',
  class: 'btn btn-large',
  text: 'Login'
})
`

To avoid these errors, the documentation should clarify the correct approach, since 'class' is a reserved word. For example:
`
<button type="{{ type || 'submit' }}" class="{{ $props.get('class') }}">
  {{ $props.get('text') }}
</button>
`

This can now be used in templates as follows, with the option to use self-closing syntax (@!component) if needed:
`
@component('components/button', {
  type: 'submit',
  class: 'btn btn-large',
  text: 'Login'
})
@end
`